src/components/display/fewlines/ConfirmationBox/ConfirmationBox.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
```

Comment:
We probably could use variant too here, don't you think? This would get rid of code duplication.